library/src/actions/lowerCase/lowerCase.ts

```diff
@@ -0,0 +1,29 @@
+import type { BaseTransformation } from '../../types/index.ts';
+
+/**
+ * Lower case transformation action type.
+ */
+export interface LowerCaseAction
+  extends BaseTransformation<string, string, never> {
+  /**
+   * The action type.
+   */
+  readonly type: 'lower_case';
+}
+
+/**
+ * Creates a lower case transformation action.
+ *
+ * @returns A lower case action.
+ */
+export function lowerCase(): LowerCaseAction {
+  return {
+    kind: 'transformation',
+    type: 'lower_case',
+    async: false,
```

Comment:
@fabian-hiller Do transforms need own type? Do transformations need introspection support?

If it is not needed, all transforms can be converted to simple wrappers:

```
export const lowerCase = transform(value => value.toLowerCase())
```

This will reduce library bundle size even more.

Reply:
I don't really know. For now, I would stick with the current approach and keep introspection support. Due to GZIP compression, I do not expect big difference in bundle size.

Reply:
Makes sense. Another question: is LowerCaseAction needed? Looks like it does not provide any benefit over using `BaseTransformation<string, string, never>`

Reply:
> Another question: is LowerCaseAction needed? Looks like it does not provide any benefit over using `BaseTransformation<string, string, never>`

I don't quite understand what you mean. Can you explain it in more detail?

Reply:
I mean, do transformation actions need a separate interface declared for them? 
`LowerCaseAction` is declared on line 6 of this file.
It adds a little bit more typing than just using `BaseTransformation<string, string, never>` directly, but I don't think it provides any useful info.

Reply:
Now I understand. Thanks for the explanation. I see your point, but for now I would continue this way so that the implementation is uniform to validation actions and to schemas.